spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -980,6 +980,29 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("Chr with null character") {
+    // test compatibility with Spark, spark supports chr(0)
+    Seq(false, true).foreach { dictionary =>
+      withSQLConf(
+        "parquet.enable.dictionary" -> dictionary.toString,
+        CometConf.COMET_CAST_ALLOW_INCOMPATIBLE.key -> "true") {
+        val table = "test0"
+        withTable(table) {
+          sql(s"create table $table(c9 int, c4 int) using parquet")
+          sql(s"insert into $table values(0, 0), (66, null), (null, 70), (null, null)")
+          val query = s"SELECT chr(c9), chr(c4) FROM $table"
+          checkSparkMaybeThrows(sql(query)) match {
+            case (None, None) => {}
+            case (Some(e), None) => throw e
+            case (None, Some(e)) => throw e
+            case (Some(e1), Some(e2)) => throw new Exception(s"$e1, $e2")
+          }
```

Comment:
This seems redundant? You already call `checkSparkAnswerAndOperator` and we do not expect any exceptions to be thrown for the input values.

Reply:
+1 , removed `checkSparkMaybeThrows`.